start.py

```diff
@@ -281,6 +288,68 @@ def main(args):
                                                          '{dir}/admin.key-pk8.pem').format(dir=TLS_CLIENT_DIR)})
                 node.put_file(CLIENT_CONF, PropertiesFile.dumps(client_properties))
 
+        if args.jwt:
+            jwt_file_path = f'{PULSAR_HOME}/jwt.token'
```

Comment:
Best to use pathlib for handling paths (doing it this way would break compatibility for Windows users, for example)

Reply:
Do not think it would be an issue here since this is a path in the Docker container so that it always gonna be unix path.